go/common/logging.go

```diff
@@ -18,8 +18,20 @@ func WarnWithID(nodeID uint64, msg string, args ...interface{}) {
 	log.Warn(fmt.Sprintf(">   Agg%d: %s", nodeID, formattedMsg))
 }
 
+// TraceWithID logs a message at TRACE level with the aggregator's identity prepended.
+func TraceWithID(nodeID uint64, msg string, args ...interface{}) {
+	formattedMsg := fmt.Sprintf(msg, args...)
+	log.Trace(fmt.Sprintf(">   Agg%d: %s", nodeID, formattedMsg))
```

Comment:
Can we pull this pattern string out into a constant?

Reply:
Done